src/cbmc/xml_interface.cpp

```
@@ -22,10 +22,9 @@ void xml_interfacet::get_xml_options(cmdlinet &cmdline)
 {
   if(cmdline.isset("xml-interface"))
   {
-    null_message_handlert message_handler;
     xmlt xml;
 
-    parse_xml(std::cin, "", message_handler, xml);
+    parse_xml(std::cin, "", null_message_handler, xml);
```

Discarding error messages here never was a good idea and should be fixed.